tests/test.mk

```diff
@@ -93,7 +93,7 @@ ifeq ($(NODE_VERSION),)
 	@echo 2>&1 "------------------------------------------------------------"
 	@echo 2>&1 "Node.js is not installed. Cannot run tests in Emscripten environment."
 	@echo 2>&1 ""
-	@echo 2>&1 "Please make sure you have \"node\" binary available in PATH and try again."
+	@echo 2>&1 "Make sure you have \"node\" binary available in PATH and try again."
```

Comment:
Rude! 😝

Reply:
Rude, but all the current style guides agree on "please" - that we should only use "please" when it's our fault and we're making the user go the extra mile to make things work due to an inconvenience/bug.

Reply:
Thanks, now I understand why.